.github/workflows/update-version.yml

```diff
@@ -0,0 +1,34 @@
+name: Update Version
+on:
+  workflow_dispatch:
+    inputs:
+      version: 
+        description: 'New version to use for the Edge Consent extension. Example: 3.0.0'
+        required: true
+
+
+      core-dependency:
+        description: '[Optional] Update Core dependency in pom.xml. Example: 3.0.0'
+        required: false
+
+jobs:
+  update-version:
+    runs-on: ubuntu-latest
+
+    steps:    
+    - name: Checkout
+      uses: actions/checkout@v3
+    
+    - name: Update Edge Consent
+      run: (./scripts/version.sh -u -v ${{ github.event.inputs.version }} -d "Core ${{ github.event.inputs.core-dependency }}")
```

Comment:
As Edge Consent defines Edge as a dependency, you'll need to include Edge here when updating the versions.
First, include the Edge version as an optional input:
```
     edge-dependency:
        description: '[Optional] Update Edge dependency in pom.xml. Example: 3.0.0'
        required: false
```
Then add Edge to the run command:
```
run: (./scripts/version.sh -u -v ${{ github.event.inputs.version }} -d "Core ${{ github.event.inputs.core-dependency }}, Edge ${{ github.event.inputs.edge-dependency }}")
```